push-images.sh

```diff
@@ -26,8 +26,9 @@ latest_version=${versions[${#versions[@]}-1]}
 for v in "${versions[@]}"; do
   if [ -z "${version}" ] || [ "${version}" == "${v}" ]; then
     # read full version from Dockerfile
-    full_version=$(grep "ARG JANUS_VERSION" ${v}/Dockerfile | cut -d"=" -f 2)
+    full_version=$(grep "ARG JANUS_VERSION" ${v}/Dockerfile | cut -d"=" -f 2 | head -n 1)
```

Comment:
Don't we have [the same problem](https://github.com/JanusGraph/janusgraph-docker/blob/480e44bd57657d114e01c5df75a13d334607ed3d/build-images.sh#L32) in `build-images.sh`?

Reply:
It is already fixed in build-images.sh

Reply:
D'OH, I just looked at the end of the line and therefore didn't see that it's placed there before the `cut`.